patroni/ctl.py

```diff
@@ -515,7 +521,12 @@ def reload(obj, cluster_name, member_names, force, role):
 def restart(obj, cluster_name, member_names, force, role, p_any, scheduled, version, pending, timeout):
     cluster = get_dcs(obj, cluster_name).get_cluster()
 
-    members = get_members(cluster, cluster_name, member_names, role, force, 'restart')
+    if scheduled is None and not force:
+        scheduled = click.prompt('When should the restart take place (e.g. 2015-10-01T14:30) ', type=str, default='now')
```

Comment:
If we started moving this line, I would really like to make it dynamic, `%YYYY-%MM-%DDT%HH:%mm`, where %HH is set to +1 hour from the current time.